serialization/unmarshaller/notification-listen.js

```diff
@@ -0,0 +1,10 @@
+module.exports = function unmarshalNotificationListen (header) {
+  var namespaces = header[0]
+
+  if (!Array.isArray(namespaces)) throw new Error('Invalid NOTIFICATION_LISTEN message header (namespaces).')
+  namespaces.map(function (namespace) {
```

Comment:
We should probably use `forEach`, instead of `map`, as we're not actually intending to build a new array.